solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/norm/Normalizer.java

```diff
@@ -31,43 +32,61 @@
  * will be received by the model.
  *
  * @see IdentityNormalizer
+ * @see MinMaxNormalizer
```

Comment:
Was needed as far as I recall for the @see cross-reference here. Alternatively, instead of adding the third see for completeness, how about removing the existing two sees?